src/config.h

```diff
@@ -98,7 +100,8 @@ const std::map<std::string, ConfigKey> CONFIG_KEY_MAP = {
   { "missing_probes", ConfigKeyMissingProbes::default_ },
   { "print_maps_on_exit", ConfigKeyBool::print_maps_on_exit },
   { "use_blazesym", ConfigKeyBool::use_blazesym },
-  { "unstable_map_decl", ConfigKeyBool::unstable_map_decl },
+  { std::string{ UNSTABLE_PREFIX } + "map_decl",
```

Comment:
This makes the key un-greppable. I think it's probably ok to just rely on convention? Once we get the unstable features really going it'll be hard to mess up (due to copy paste effect)